src/main/java/org/folio/rest/impl/PoLinesAPI.java

```diff
@@ -33,7 +34,7 @@ public void getOrdersStoragePoLines(String query, int offset, int limit, String
       Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     vertxContext.runOnContext((Void v) -> {
       EntitiesMetadataHolder<PoLine, PoLineCollection> entitiesMetadataHolder = new EntitiesMetadataHolder<>(PoLine.class, PoLineCollection.class, GetOrdersStoragePoLinesResponse.class);
```

Comment:
According to [Acquisitions API Listing](https://docs.google.com/spreadsheets/d/1se-a2owRfHLG5eaAZglx4vLvRClCKGp2wZCfD39ZJRY/view#gid=894044560&range=B48) the `/orders-storage/order-lines` endpoint is going to use view

Reply:
oh right Thanks, I missed that part